akka-actor-typed-tests/src/test/scala/docs/akka/typed/IntroSpec.scala

```diff
@@ -186,7 +186,7 @@ object IntroSpec {
             handle ! PostMessage("Hello World!")
             Behaviors.same
           case MessagePosted(screenName, message) =>
-            context.log.info("message has been posted by '{}': {}", Array(screenName, message))
+            context.log.info("message has been posted by '{}': {}", Array(screenName, message): _*)
```

Comment:
Why create the array at all, doesn't this work:
`context.log.info("message has been posted by '{}': {}", screenName, message)`

Reply:
This is the only way I found out to tell the compiler we had a specific signature. Maybe there's something better. That I don't know by now.

Reply:
Oh, that's pretty bad that we get that from Scala.